client/components/home.jsx

```diff
@@ -61,17 +62,20 @@ export class Home extends React.Component {
           <h1 style={styles.header}> Explore </h1>
           <p style={styles.subText}> There are so many great beers around the world. Sometimes it can be hard to keep track of all the different kinds! Progressive Beer is a handy web app that is designed to help you learn everything there is to know about beers! Explore the many beer styles in list below for more information. </p>
 
-          <div style={styles.search}>
-            <TextField floatingLabelText="Filter beer styles..." /> <SearchIcon />
-          </div>
+          <AboveTheFoldOnlyServerRender skip={true}>
+            <div>
+              <div style={styles.search}>
+                <TextField floatingLabelText="Filter beer styles..." /> <SearchIcon />
+              </div>
+              <div style={styles.root}>
+                <GridList cellHeight={180} style={styles.gridList}>
+                  <BeerCard />
```

Comment:
this means all of the beer-cards will be CSR

is the shell expected to be like this?
```
<head />
<body>
  <some random divs>
  <footer />
</body>
```